src/__tests__/components/SelectFioAddress.test.tsx

```diff
@@ -30,7 +30,7 @@ describe('SelectFioAddress', () => {
           payee_fio_address: 'fio@fio',
           payer_fio_address: 'fio2@fio',
           payer_fio_public_key: '123123',
-          status: 'pending',
```

Comment:
Why did this change?

Reply:
Since we typed this as `string`, there was nothing to prevent us from making mistakes. According to the docs, "pending" is not a valid value for this field. So, I picked the actually valid value that had the same meaning.